src/kirby/components/toolbar/toolbar.component.tns.html

```diff
@@ -0,0 +1,27 @@
+<ActionBar class="toolbar {{themeColor}}">
```

Comment:
I have a really hard time reading this. Is it possible for you to someway make it more simple/readable?